src/5e-SRD-Levels.json

```diff
@@ -8158,5 +8154,974 @@
     },
     "url": "/api/subclasses/lore/levels/14",
     "index": 247
+  },
+  {
+    "level": 1,
+    "feature_choices": [],
+    "features": [
+      {
+        "name": "Bonus Proficiency",
+        "url": "/api/features/bonus-proficiency"
```

Comment:
I hope this doesn't bite us in the ass later. But it's fine for now.

Reply:
What do you mean, exactly?

Reply:
If we ever have another class with Bonus Proficiency but it's not say "Heavy Armor". Luckily Bard has a similar feature but it's called "Bonus Proficiencies".

Reply:
Ah I see. I didn't add any features in this PR, only used existing ones. Perhaps one for a separate issue?